code/modules/spells/targeted/ethereal_jaunt.dm

```diff
@@ -28,12 +29,12 @@
 	if(targets.len > 1)
 		mass_jaunt(targets, duration, enteranim, exitanim, mist)
 	else
-		ethereal_jaunt(targets[1], duration, enteranim, exitanim, mist)
+		ethereal_jaunt(targets[1], duration, enteranim, exitanim, mist, movement_tally)
 
-/proc/ethereal_jaunt(var/mob/living/target, duration, enteranim = "liquify", exitanim = "reappear", mist = 1)
+/proc/ethereal_jaunt(var/mob/living/target, duration, enteranim = "liquify", exitanim = "reappear", mist = 1, movement_tally)
 	var/mobloc = get_turf(target)
 	var/previncorp = target.incorporeal_move //This shouldn't ever matter under usual circumstances
-	if(target.incorporeal_move == INCORPOREAL_ETHEREAL) //they're already jaunting, we have another fix for this but this is sane
```

Comment:
Check if this define is still being used anywhere, if not, remove it.

Reply:
Or you can just avoid magic numbers and make a new define for 1 and leave this as define in the child spells.

Reply:
Used by mass jaunt, which I didn't touch
I swapped incorporeal_move with a variable named movement_tally that can be var-edited for further spells. 
While searching for instances of incorporeal_ethereal I have found an interaction for it for areas that do not allow ethereal jaunting, and I will fix this when I get around to do it